### ISSUE TITLE
MT3D-USG - GWV8

### ISSUE THREAD

**Issue by sdevons** [MODFLOW-USGS/mt3d-usgs#88]
I've tried running MT3D-USG in Groundwater Vistas 8 and it comes up with the error 'run-time error F29: Open(unit 26)'. As an experiment I then did the tutorial that comes with the software and followed the manual exactly and I still got the error so it doesn't seem like it's related to my model. Does anybody know what the error is? Thanks

**Comment by emorway-usgs** [comment]
Hello @sdevons, Have you tried running your model outside of GW Vistas?  That is, from the command line?  If you're able to zip up your model, and maybe reduce it to just the first stress period to keep file size down, and post it to this issue that would be helpful for us to determine if it is an MT3D-USGS issue, or a Groundwater Vistas issue.

And for clarity, I just want to make sure you're not referring to a MODFLOW-**USG** problem?  The subject line reads "MT3D-USG", not MT3D-USGS, so just want to make sure we're discussing the same model.

**Comment by kcognac** [comment]
I had the same issue as sdevons using USG Transport in GW Vistas 8.  I resolved it by manually deleting all Vistas generated input files in the root folder and recreating them (with Vistas).  The error seemed to indicate that an input file was open elsewhere even though it wasn't.

**Comment by emorway-usgs** [comment]
Thanks for the suggestion, @kcognac.  Having never heard back from sdevons, closing this issue.

**Comment by bigflat** [comment]
Hello, I came across this, as I have been having the same issue in GWV8. Did anyone figure it out?

**Comment by emorway-usgs** [comment]
Hello @bigflat, Can you clarify if you're trying to use MT3D-USG**S** or transport inside MODFLOW-USG?  The subject line reads MT3D-USG which is a bit unclear.  Also, did you see how @kcognac resolved this, did that not work for you?

**Comment by bigflat** [comment]
Thanks for the quick response @emorway-usgs. Sorry, no, I am not running MODFLOW-USG, I missed that, I just saw the same error that I got. I am running MT3D-USGS through Groundwater Vistas 8. I didn't fully delete and recreate files, but I did go into a (supposedly working) example shipped with the software from ESI (the first one in the tutorial manual) and try to run it. The error above went away, but a new error came up, "time steps or stress periods not found in files." But perhaps here is not the place to deal with this. I also reached out to the GWV folks.
